ansible/roles/common/tasks/main.yml

```
@@ -0,0 +1,136 @@
+    value: "{{ item.value }}"
+  become: yes
+  loop:
+    - { type: 'soft', item: 'nofile', value: '65536' }
+    - { type: 'hard', item: 'nofile', value: '65536' }
+  tags: common
+
+- name: Add project user to sudo group
+  user:
+    name: "{{ project_user }}"
+    groups: sudo
+    append: yes
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)